server/data/userClient.ts

```diff
@@ -27,6 +27,10 @@ export default class UserClient {
     return (await this.restClient.get({ path: paths.users.profile({}) })) as User
   }
 
+  async getUserList(): Promise<Array<User>> {
+    return this.restClient.get({ path: paths.users.index({}) }) as Promise<Array<User>>
+  }
+
```

Comment:
hmm, I don't think we want the list of all the users (applicants included) do we? 

If we pass in a list of roles I think we can limit this list to staff members to whom tasks can be allocated? e.g.  `roles = assessor, matcher`, which might be a lot shorter and more relevant?

Reply:
Sorted! 👍